src/services/accounts.js

```diff
@@ -301,7 +255,7 @@ async function checkAccountStatus({ res, auth }) {
  * @param {String} institution
  * @param {String} useCase
  */
-const updateAccountData = async ({ userId, email, shortname, institution, useCase }) => {
+const updateAccountData = async ({ userId, email, shortname, institution, useCase, role }) => {
```

Comment:
If you want to reuse this for both users and admins, you need to make sure that: 1. normal users can only change their own data, 2. only admins can change roles (and emails, for now)